settings.gradle.kts

```diff
@@ -19,6 +19,7 @@ include(
 
 dependencyResolutionManagement {
   repositories {
+    mavenLocal()
```

Comment:
Remove this? In general it's not good to keep this around, as it can conflict with what's on the snapshot repo (picking up the wrong packages).

Reply:
> How is the intended workflow with locally installed maven artifacts?

For the time being this: https://github.com/restatedev/e2e#how-to-test-java-sdk-changes Although we could reconsider it later. In any case, I think the mavenLocal shouldn't be enabled by default, but rather be opt-in, as it's easy to fall in trap with it.

Reply:
Alright, I will remove it.